example/color_maps_sf_example.cpp

```diff
@@ -580,7 +580,7 @@ int main(int argc, char** argv)
     if (debug)
     {
        for (int64_t i = image_width / 2; i < image_width; ++i)
-          points[image_width * (image_height - 1) + i] = i & 1 ? 1 : 0;
+          points[image_width * (image_height - 1) + i] = i & 1;
```

Comment:
This is a superfluous change IMO.

Reply:
The expression is superfluous to start with

Reply:
How so?

Reply:
i & 1 can only evaluate to 1 or 0, so the ternary operator is redundant

Reply:
Doh! My apologies I read the diff backwards and thought you were adding the ?: operator.

My bad again!